node/network/approval-distribution/src/lib.rs

```diff
@@ -138,9 +138,9 @@ impl AggressionConfig {
 impl Default for AggressionConfig {
 	fn default() -> Self {
 		AggressionConfig {
-			l1_threshold: Some(10),
-			l2_threshold: Some(25),
-			resend_unfinalized_period: Some(5),
+			l1_threshold: Some(13),
```

Comment:
Why do we bump by 3 , when we only expect finality lag to increase by one slot ?

Reply:
I was just being cautious and wanted to make double sure that escalation does not trigger, when there is no reason to.